pyuvdata/uvdata/mwa_corr_fits.py

```diff
@@ -168,6 +171,10 @@ def read_mwa_corr_fits(
         """
         Read in MWA correlator gpu box files.
 
+        Default settings divide out the digital gains and the coarse band shape.
+        If the desired output is raw correlator data, set remove_dig_gains=False
+        and remove_coarse_band=False.
```

Comment:
Maybe add something to the first sentence to explain why -- this is coming from something Lindsay said about trying to help new users decide whether they want to use the defaults or change them. Maybe something like "to remove some of the instrumental effects in the bandpass".

I think for truly raw data you'd also want to make sure that `correct_cable_len` and `phase_to_pointing_center` are False.

Reply:
How's this: 
```
        The default settings remove some of the instrumental effects in the bandpass
        by dividing out the digital gains and the coarse band shape. 
        If the desired output is raw correlator data, set remove_dig_gains=False,
        remove_coarse_band=False, correct_cable_len=False, and 
        phase_to_pointing_center=False.
```

Reply:
```
Or something like this, and I can change it if we change the default behavior:
The default settings remove some of the instrumental effects in the bandpass
        by dividing out the digital gains and the coarse band shape. 
        If the desired output is raw correlator data, set remove_dig_gains=False and
        remove_coarse_band=False. Additionally, the default settings correct_cable_len=False and phase_to_pointing_center=False are required.
```

Reply:
I like the first one, I think it's perfect.

Reply:
Done!